packages/library-base/src/commonMain/kotlin/io/realm/kotlin/internal/dynamic/DynamicMutableRealmImpl.kt

```diff
@@ -33,7 +33,7 @@ import io.realm.kotlin.internal.runIfManaged
 import io.realm.kotlin.internal.toRealmObject
 import io.realm.kotlin.query.RealmQuery
 
-internal open class DynamicMutableRealmImpl(
+public open class DynamicMutableRealmImpl(
```

Comment:
I always found it nice with a comment that tracked why we exposed those internal classes. Maybe really obvious here, but ...